jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml

```diff
@@ -587,27 +588,23 @@
     trigger-job: ''
     timeout: 240
     jenkins_node: 'master'
```

Comment:
good catch. that's one of the main reasons to do this change in the first place. :)